internal/handlers/handlers.go

```diff
@@ -419,3 +422,86 @@ func (h *Handlers) DeletionURLs(w http.ResponseWriter, r *http.Request) {
 func (h *Handlers) ResultBody(res string) string {
 	return h.baseURL + "/" + res
 }
+
+// GetStatus docs
+// @Tags GET
+// @Summary Get count urls and users in DB
+// @Description Get count urls and users in DB
+// @Security ApiKeyAuth
+// @Accept json
+// @Produce json
+// @Success 200 "OK"
+// @Failure 403 "Forbidden"
+// @Failure 500 "Internal server error"
+// @Router /api/internal/status [get]
+// GetStatus собирает сколько urls в базе и сколько users.
+func (h *Handlers) GetStatus(w http.ResponseWriter, r *http.Request) {
+
+	// проверяем есть ли доверительный IP
+	if h.trustedSubnets == "" {
+		w.WriteHeader(http.StatusForbidden)
+		return
+	}
+
+	// проверяем доверительный IP
+	err := checkIP(r, h.trustedSubnets)
+	if err != nil {
+		h.logger.Error("error = ", logger.ErrAttr(err))
+		w.WriteHeader(http.StatusForbidden)
+		return
+	}
+
+	// получаем сумму всех urls и users в базе
+	countURLs, countUsers, err := h.service.GetCountURLsAndUsers()
+
+	if err != nil {
+		h.logger.Error("error = ", logger.ErrAttr(err))
+		w.WriteHeader(http.StatusInternalServerError)
+		return
+	}
+
+	w.Header().Set("Content-Type", "application/json")
+	w.WriteHeader(http.StatusOK)
+
+	if err = json.NewEncoder(w).Encode(map[string]int{"urls": countURLs, "users": countUsers}); err != nil {
+		h.logger.Error(`"error": "failed to marshal response", "details": `, logger.ErrAttr(err))
```

Comment:
Мне кажется, что такая структура для сообщения логера избыточная. Если логер сконфигурирован корректно, то он сам сформирует корректную структуру.
```suggestion
		h.logger.Error("failed to marshal response" , logger.ErrAttr(err))
```

Reply:
исправил